packages/firestore/CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Unreleased
+- [changed] Fixed a crash on iOS 13 that occurred when persistence was enabled
+  in a background tab (#2232).
+- [fixed] Fixed an issue in the interaction with the Firestore Emulator that
+  caused requests with timestamps to fail.
```

Comment:
I rewrote this slightly when I addressed the merge conflict (to add a missing "the").

cc @yuchenshi